spec/enumerable_spec.rb

```diff
@@ -24,27 +25,32 @@
     end
 
     context "passes :tokenizer through to the index" do
-      before(:each) { @letter_count_tokenizer = lambda { |str| str.size.to_s } }
+
+      before(:each) {
```

Comment:
The G2 convention for blocks is to only use the curlies to denote a single-line block; For multiline, we use `do` .. `end`